src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -642,7 +642,7 @@ public async Task OpenDocumentAsync(Uri documentUri, string? text = null, string
                 {
                     // LSP open files don't care about the project context, just the file contents with the URI.
                     // So pick any of the linked documents to get the text from.
-                    var sourceText = await TestWorkspace.CurrentSolution.GetDocuments(documentUri).First().GetTextAsync(CancellationToken.None).ConfigureAwait(false);
+                    var sourceText = (await TestWorkspace.CurrentSolution.GetDocumentsAsync(documentUri, CancellationToken.None)).First().GetTextAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
nit.  my preference is to break into statements.  but this is also tests, so i don't care.